.github/workflows/docker.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           context: .
           file: ./build/dockerfiles/event_watcher.Dockerfile
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64
```

Comment:
similar to `coordinator-api`, now rollup-relayer, gas-oracle, and event-watcher have dependencies to .so files, thus removing the `linux/arm64` platform docker image.